shared/fs/container.js

```diff
@@ -26,7 +26,9 @@ const mapStateToProps = (state: TypedState, {routeProps}: OwnProps) => {
   }
 }
 
-const mapDispatchToProps = (dispatch: Dispatch) => ({})
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  _loadFolderList: path => Constants.dispatchFolderListLoad(dispatch, Types.stringToPath(path)),
```

Comment:
path here is already a Types.Path and not a string
Also method created in mapDispatchToProps need typing of params as the typing is kinda weak here (visible if you do a flow coverage on this file) so make it
```
(path: Types.Path) => dispatch()...
```